### ISSUE TITLE
Don’t attempt to update Twitter feeds for local and iCloud accounts

### ISSUE THREAD

**Issue by brentsimmons** [Ranchero-Software/NetNewsWire#3851]
If it’s a Twitter feed, then don’t even attempt to update it the normal way (or any way at all). These feeds should just be inert.

(Twitter feeds are feeds where the hostname == `twitter.com` — note that `blog.twitter.com`, for instance, could be the hostname for an actual RSS feed.)

**Comment by brentsimmons** [comment]
I think this has been done — @vincode-io true? Let’s close this if so.

**Comment by vincode-io** [comment]
Yeah, the Twitter Extension code has been completely been removed. It will always receive a 404 for the old Twitter Extension feeds.

Did you want us to filter these out of the the feed fetcher as well?

**Comment by vincode-io** [comment]
Now that I think about it, that is probably exactly what you meant. M

**Comment by brentsimmons** [comment]
Yup — we shouldn’t even try to fetch those feeds at all, knowing it won’t work. Save the battery/networking/whatever use.

**Comment by ashishgandhi** [comment]
Now that there is no Twitter support, can this be closed?
